src/test/sighash_tests.cpp

```diff
@@ -93,27 +93,26 @@ void static RandomScript(CScript &script) {
         script << oplist[insecure_rand() % (sizeof(oplist)/sizeof(oplist[0]))];
 }
 
+// Overwinter tx version numbers are selected randomly from current version range.
+// http://en.cppreference.com/w/cpp/numeric/random/uniform_int_distribution
+// https://stackoverflow.com/a/19728404
+std::random_device rd;
+std::mt19937 rng(rd());
+std::uniform_int_distribution<int> verdis(CTransaction::OVERWINTER_MIN_CURRENT_VERSION,
```

Comment:
``version_dist`` or ``version_distribution`` would be better. Does not block.

Reply:
changing to version_dist